circuits/spartan/package.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "@anonklub/spartan-ecdsa",
+    "version": "1.0.0",
+    "description": "",
+    "main": "index.js",
+    "type": "module",
+    "scripts": {
+        "check-wasm-pack": "command -v wasm-pack >/dev/null 2>&1 || echo \"wasm-pack is not installed\"\n",
+        "compile": "pnpm run check-wasm-pack && wasm-pack build . --target bundler --out-dir wasm-pkg --out-name index --scope anonklub"
+    },
+    "keywords": [],
+    "author": "",
+    "license": "ISC",
+    "dependencies": {
+        "pako": "^2.1.0",
+        "spartan-ecdsa-circuits": "^1.0.0"
+    },
+    "devDependencies": {
+        "ts-node": "^10.9.1",
+        "typescript": "5.2.2"
+    }
+}
```

Comment:
The reason is that `package.json` files are ignored in the `.prettierignore` should we remove it?

Reply:
yeah I noticed it yesterday. Sorry I should have told you.
I still want to exclude package.json files of the packages that are to be published from format check because `changeset` will edit them (change the version) upon release and messes with existing prettier format.
For consistency then I decided to exclude all package.json from the check.
Let's keep it that way.

Ignore my comment.
I still do like not see the github warning about newline lol. So I do this manually atm.

The right solution would be to define a [prettier override](https://prettier.io/docs/en/configuration.html#configuration-overrides).
But let's do this later

Reply:
capture: https://github.com/anonklub/anonklub/issues/297